FirebaseAuth/Sources/Swift/SystemService/AuthStoredUserManager.swift

```diff
@@ -36,14 +36,17 @@ import Foundation
   /// The key to encode and decode the stored user.
   private static let storedUserCoderKey = "firebase_auth_stored_user_coder_key"
 
+  // TODO: Should keychainServices be AuthStorage
```

Comment:
I'd say the answer depends on what kind of test coverage we want for this class. It'd be convenient and make things more consistent but it would require some extra work that might not be worth it. If you take a look at the API of `AuthSharedKeychainServices`, it doesn't easily conform to the protocol API of `AuthStorage`. We'd need to rework this class and the clients that call it. 

Do the class's unit tests reach codepaths that touch the keychain?

Reply:
got it, thanks. yeah, I think disabling for now sgtm.